android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -451,6 +451,18 @@ public void setTermsOfServiceUrl(final String urlString) {}
 
      // Data Passing
 
+    @ReactMethod()
+    public void setUserSegmentField(final String name)
```

Comment:
I think it's fine to just name this `setUserSegment(...)`

Reply:
renamed to setUserSegment